addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2992,19 +2994,42 @@ var SnippetsMenu = Widget.extend({
                 break;
         }
 
+        this.options.wysiwyg.toolbar.el.classList.remove('oe-floating');
+
+        // Create toolbar custom container
         const $customizeBlock = $('<WE-CUSTOMIZEBLOCK-OPTIONS id="o_we_editor_toolbar_container"/>');
-        const $title = $("<we-title><span>" + titleText + "</span></we-title>");
+        const $header = $("<we-title/>");
 
-        $customizeBlock.append($title);
+        $header.append($("<span>" + titleText + "</span>"));
+        $customizeBlock.append($header);
         $customizeBlock.append(this.options.wysiwyg.toolbar.$el);
         $(this.customizePanel).append($customizeBlock);
+
+        // Create table-options custom container
+        const $customizeTableBlock = $('<WE-CUSTOMIZEBLOCK-OPTIONS id="o_we_editor_table_container"/>');
+        const $tableHeader = $("<we-title/>");
+        const $tableOptionsList = $('#oe-table-options');
+
+        $tableOptionsList.removeClass('d-none');
+        $tableHeader.append($("<span>" + titleTableText + "</span>"));
+
+        $customizeTableBlock.append($tableHeader);
+        $customizeTableBlock.append($tableOptionsList);
+        $(this.customizePanel).append($customizeTableBlock);
+
+        // Handle we-title button, cleanup html
+        $header.append(document.getElementById('removeFormat'));
+        $tableHeader.append(document.getElementById('oe-table-delete-table'));
+        $('#table').remove();
```

Comment:
In fact this PR is really more something to see with the editor team, I can check the scss etc but this whole JS I don't really know it.